network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,87 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-improved-lokistack-integration"]
+==== Improvements for Network Observability without Loki
+You can create Prometheus alerts for the *Netobserv* dashboard using DNS, Packet drop, and RTT metrics, even if you don't use Loki. In the previous version of Network Observability, 1.4, these metrics were only available for querying and analysis in the *Network Traffic*, *Overview*, and *Topology* views, which are not available without Loki. For more information, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-zones"]
+==== Availability zones
+You can configure the `FlowCollector` resource to collect information about the cluster availability zones. This configuration enriches the network flow data with the link:https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone[`topology.kubernetes.io/zone`] label value applied to the nodes. For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-zonesnw-observe-network-traffic[Working with availability zones].
+
+[id="network-observability-enhanced-configuration-and-ui-1.5"]
+==== Notable enhancements 
+The 1.5 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration.
+
+[discrete]
+[id="web-console-enhancements-1.5_{context}"]
+===== Web console enhancements:
+
+* There are new panels added to the *Overview* view for DNS and RTT: Min, Max, P90, P99.
+* There are new panel display options added:
+** Focus on one panel while keeping others viewable but with smaller focus. 
+** Switch graph type. 
+** Show *Top* and *Overall*.
+* A collection latency warning is shown in the *Custom time range* pop-up window.
+* There is enhanced visibility for the contents of the *Manage panels* and *Manage columns* pop-up windows.
+* The Differentiated Services Code Point (DSCP) field for egress QoS is available for filtering QoS DSCP in the web console *Network Traffic* page.
+
+[discrete]
+[id="configuration-enhancements-1.5_{context}"]
+===== Configuration enhancements:
+
+* The `LokiStack` mode in the `spec.loki.mode` specification simplifies installation by automatically setting URLs, TLS, cluster roles and a cluster role binding, as well as the `authToken` value. The `Manual` mode allows more control over configuration of these settings.
+* The `spec.agent.ebpf.kafkaBatchSize` default is changed from `10MB` to `1MB` to enhance performance when Kafka is being used. 
```

Comment:
@jotak re: https://github.com/netobserv/network-observability-operator/pull/566

wdyt? Keep it here? Move it elsewhere?

Reply:
I like the placement, one tweak I might make: 
`to enhance eBPF performance when Kafka is being used.`

Reply:
I think it should be more visible, and clearly call for user action, because users upgrading won't have this set automatically. I would move it as the first thing in notable enhancements, and write something like:
"When upgrading from an existing installation, this new value will not be set automatically in your configuration. If you monitor a performance regression with the eBPF Agent memory consumption after upgrading, you might consider reducing the `kafkaBatchSize` value.

Reply:
ah sorry I just realize there is an "important" section below, my bad

Reply:
Addressed! TY!